packages/liveblocks-react/src/room.tsx

```diff
@@ -443,7 +443,9 @@ export function createRoomContext<
             >
           }
         >
-          {props.children}
+          <CommentsRoomProvider room={room}>
+            {props.children}
+          </CommentsRoomProvider>
```

Comment:
It seems like we don't necessarily need this additional layer and could do `void getThreadsUpdates(room.id);` here instead, next to connecting to rooms, etc.